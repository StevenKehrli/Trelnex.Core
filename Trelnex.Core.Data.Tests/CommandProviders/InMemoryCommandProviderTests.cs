namespace Trelnex.Core.Data.Tests.CommandProviders;

public class InMemoryCommandProviderTests : CommandProviderTests
{
    [SetUp]
    public async Task TestFixtureSetup()
    {
        // create our command provider
        var factory = await InMemoryCommandProviderFactory.Create();

        _commandProvider =
            factory.Create<ITestItem, TestItem>(
                typeName: "test-item",
                TestItem.Validator,
                CommandOperations.All);
    }
}
