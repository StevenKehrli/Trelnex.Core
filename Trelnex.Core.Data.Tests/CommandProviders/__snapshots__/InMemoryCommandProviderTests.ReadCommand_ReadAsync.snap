{
  "Item": {
    "Id": "a8cf4bc4-745a-471c-8fb1-5d4e124bbde2",
    "PartitionKey": "2b541bfd-4605-48f9-b1bc-5aba5f64cd24",
    "TypeName": "test-item",
    "CreatedDate": "2025-01-07T00:54:42.113127Z",
    "UpdatedDate": "2025-01-07T00:54:42.113127Z",
    "DeletedDate": null,
    "IsDeleted": null,
    "ETag": "57c8d5d5-f6d0-49fe-b55f-4e59f872ced4",
    "Message": "Message #1"
  }
}
